index.template.html

```diff
@@ -3916,6 +3925,115 @@ <h5>Inferencing of Personally Identifiable Information Privacy Risk</h5>
+      
+    <p>
+    A <em>Thing Model</em> is a special kind of a <a>Thing Description</a> with less restriction as 
+    driven in <a href="#sec-vocabulary-definition"></a> and <a href="#sec-td-serialization"></a>. Thing Model samples can 
+    also be seen as a subset of the <a>Thing Descriptions</a> information model, which may not contain, 
+    for example, concrete protocol and security information.
+    </p>
+    <p>
+    The Thing Model enables:
+    <ul>
+    <li> onboard and manage of multiple <a>Thing</a> models, e.g., by a cloud service. </li>
+    <li> simulate of devices/<a>Things</a> that have not yet been developed.</li>
+    <li> develop common applications across devices from different manufacturers that share a common <a>Thing</a> model.</li>
+    <li> combining multiple models into a <a>Thing</a>.
+    </ul>
+    </p>
+    <p>
+    The Thing Model is a logical description of the interface and possible interaction with <a>Thing's</a>
+    <a>Properties</a>, <a>Actions</a>, and <a>Events</a>, however it does not contain <a>Thing</a> instance-specific
+    information, such as security information and concrete protocol usage, or even a serial number and GPS location, .
+    </p><p>
+    Since a Thing Model MAY not contain complete <a>Protocol Binding</a> information such as endpoint address and/or
+    specific security mechanism, the <em>forms</em>, <em>securityDefinitions</em>, and 
+    <em>security</em> terms and its sub-terms MAY not be present.
+    </p>
+
+    <p>    
+    In the following there is a sample Thing Model for a lamp without any protocol and security information.
+    </p>
+    
+    <pre class="example" title="MyLampThingTemplate serialized in JSON">
+    {
+        "@context": ["https://www.w3.org/2019/wot/td/v1"], 
+        "@type" : "ThingTemplate",
+        "title": "Lamp Thing Description",
+        "description" : "Lamp Thing Description Template",
+        "properties": {
+            "status": {
+                "description" : "current status of the lamp (on|off)",
+                "type": "string",
+                "readOnly": true
+            }
+        },
+        "actions": {
+            "toggle": {
+                "description" : "Turn the lamp on or off"
+            }
+        },
+        "events": {
+            "overheating": {
+                "description" : "Lamp reaches a critical temperature (overheating)",
+                "data": {"type": "string"}
+            }
+        }
+    }
+    </pre>
+    
+    
+    
+    <p>
+     The same Thing Model can be implemented by <a>Things</a> from multiple vendors,
+    a <a>Thing</a> can implement multiple Thing Model, define additional metadata
+    (vendor, location, security) and define bindings to concrete protocols.
+    To avoid conflicts between properties, actions and events from different Thing Model
+    that are combined into a common <a>Thing</a>, all these identifiers must be unique within a <a>Thing</a>.
+    </p><p>
```

Comment:
Uniqueness across different providers of thing models is difficult to achieve. IMHO we need a namespace concept.

Reply:
good point. The idea would be to use the "$ref" mechanism from JSON Schema. I will check how namespace conflicts are handled.